modules/ROOT/pages/dos-policy.adoc

```diff
@@ -15,7 +20,7 @@ This source is IP based. If an attacker spoofs the source IP address, these meas
 | *Protocol Errors*
 | When a client sends malformed protocol messages.
 
-A CAP policy violation escalates directly as a protocol error.
+An HTTP Limits policy violation escalates directly as a protocol error.
```

Comment:
The word escalate is better because this is how the security works. Things get escalate. If condition A is met, then it escalates to B. It's moving up a level. I think that concept is understood in the security world in terms of DoS.